packages/core/src/browser/window/window-title-service.ts

```diff
@@ -100,7 +100,7 @@ export class WindowTitleService {
         if (developmentHost) {
             this._title = developmentHost + this.separator + this._title;
         }
-        document.title = this._title;
+        document.title = !!this._title ? this._title : FrontendApplicationConfigProvider.get().applicationName;
```

Comment:
I believe this is effectively the same as:
```suggestion
        document.title = this._title || FrontendApplicationConfigProvider.get().applicationName;
```